Tests/UITests/TestAppUITests/FirebaseAccountTests.swift

```diff
@@ -392,10 +392,10 @@ extension XCUIApplication {
         XCTAssertTrue(staticTexts["Please fill out the details below to create a new account."].waitForExistence(timeout: 6.0))
         sleep(2)
 
-        try textFields["E-Mail Address"].enter(value: username)
+        try collectionViews.textFields["E-Mail Address"].enter(value: username)
```

Comment:
Interesting that it doesn't find it in the main text fields which should be a global catch-all 🤔

Reply:
It's because we render signup as a sheet now. And behind the sheet as the AccountSetup view with the "E-Mail Address" and "Password" login fields. So we have to be more specific in our selection.

Reply:
Ah, makes sense 👍